### PR TITLE
Reference table mapping

### PR DESCRIPTION
**Pull request by l0gesh29**

### CODE REVIEW

File: bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py [hunk 1]
```diff
@@ -124,7 +138,6 @@ def set_supplier(source_doc,target_doc,source_parent):
 			"doctype": "Outward Bank Payment",
 			"field_map": {
 				"supplier": "party",
-				"grand_total": "amount",
```
Comment (Vigneshsekar) [inline bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py:127]: Why did you remove this?

File: bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py [hunk 2]
```diff
@@ -96,15 +99,20 @@ def make_bank_payment(source_name, target_doc=None):
...
+			"outstanding_amount":source_doc.outstanding_amount
+		})
 	from frappe.model.mapper import get_mapped_doc
 	doclist = get_mapped_doc("Purchase Invoice", source_name,{
 		"Purchase Invoice": {
 			"postprocess": set_supplier,
 			"doctype": "Outward Bank Payment",
 			"field_map": {
 				"supplier": "party",
-				"outstanding_amount": "amount",
```
Comment (Vigneshsekar) [inline bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py:107]: Why did you remove this?

**Review by Vigneshsekar — CHANGES_REQUESTED** [review]

File: bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py (hunk 1 above)
Comment (l0gesh29) [reply bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py:127]: I thought, for manual reconciliation, the user will enter the total amount as per the allocated amounts in the below payment reference table

File: bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py (hunk 2 above)
Comment (l0gesh29) [reply bank_api_integration/bank_api_integration/doctype/outward_bank_payment/outward_bank_payment.py:107]: The above-mentioned statement is also applicable for this condition.

**Review by Vigneshsekar — APPROVED** [review]
